src/iov.c

```diff
@@ -37,8 +37,8 @@
 #include <ofi.h>
 #include <ofi_iov.h>
 
-uint64_t ofi_copy_iov_buf(const struct iovec *iov, size_t iov_count, uint64_t iov_offset,
-			  void *buf, uint64_t bufsize, int dir)
+size_t ofi_copy_iov_buf(const struct iovec *iov, size_t iov_count, size_t iov_offset,
+                        void *buf, size_t bufsize, int dir)
 {
 	uint64_t done = 0, len;
```

Comment:
Change to `size_t` (sorry; missed this).

Reply:
No worries. New version coming.

Reply:
@darrylabbate I added one more commit to convert the remaining `uint64_t` in `src/iov.c`. I can squash before merge if it looks OK.

Reply:
I think this LGTM. Much appreciated.

Reply:
👍 should I squash the first and third commits? Anything else to do before merging?

Reply:
You can squash the first and third commits. If no one else has comments, we can merge after CI passes.